### PR TITLE
increase bootstrap token ttl

### PR DESCRIPTION
**Pull request by bullet1337**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,9 @@
 
 ------
 
+## Build
```
Comment (syndicut) [inline README.md]: I think we shouldn't add internal links to upstream README

Comment (bullet1337) [reply README.md]: but how can we link our build to the project to make it convenient?

Comment (bullet1337) [reply README.md]: I'm not sure if we are going to contribute to upstream

**Review by Intar — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bullet1337) [reply README.md]: removed

**Pull request merged**